pkg/question/question.go

```diff
@@ -642,7 +693,7 @@ func AskVpc(h *ec2helper.EC2Helper) (*string, error) {
 				vpcName = fmt.Sprintf("%s(%s)", *vpcTagName, *vpc.VpcId)
 			}
 
-			if *vpc.IsDefault {
+			if userDefaultVpcId != "" && *vpc.VpcId == userDefaultVpcId || *vpc.IsDefault && defaultOptionValue == cli.ResponseNew {
```

Comment:
The first half of this if statement makes sense to me. But can you explain why we need `&& defaultOptionValue == cli.ResponseNew` in the second half? What situation is the user in when they execute that code path?

Reply:
cli.ResponseNew is the default setting of the defaultOptionValue variable. If the defaultVpcId is already set as the deafultOption then the system default shouldn't override it.